### PR TITLE
Completion of french canadian (fr_CA) translation

### PR DESCRIPTION
**Pull request by pnault**
Hi,
These files should complete the translation.

Pierre

### CODE REVIEW

**Comment by asmecher** [comment]
See https://github.com/pkp/ojs/pull/1160 for the OJS part of this. Thanks, @pnault! Over to you, @mtub.

**Comment by asmecher** [comment]
Continuing to push to this existing PR is a good way to do it, @pnault -- github provides tools to support this kind of thing.

File: locale/fr_CA/editor.xml [hunk 1]
```diff
@@ -99,4 +99,40 @@
...
+	<message key="editor.review.readNewReview">Lire la nouvelle évaluation</message>
+	<message key="editor.review.responseDue">Réponse due pour le {$date}</message>
+	<message key="editor.review.requestSent">Requête envoyée</message>
+	<message key="editor.review.requestAccepted">Requête acceptée</message>
+	<message key="editor.review.revertDecision">Renverser la décision</message>
+	<message key="editor.review.reviewDue">Évaluation due pour le: {$date}</message>
+	<message key="editor.review.reviewConfirmed">Évaluation confirmée</message>
+	<message key="editor.review.reviewSubmitted">Évaluation soumise</message>
+	<message key="editor.submission.copyedit.manageCopyeditFilesDescription">Tous les fichiers qui ont déjà été téléchargés dans une étape de soumission peuvent être ajoutés à la """"liste des fichiers de copie""" en cochant la case Inclure ci-dessous et en cliquant sur Rechercher: tous les fichiers disponibles seront répertoriés et peuvent être choisis pour inclusion. Les fichiers supplémentaires qui ne sont pas déjà chargés sur une étape peuvent être ajoutés à la liste en cliquant sur le lien Télécharger un fichier.</message>
```
Comment (mtub) [inline locale/fr_CA/editor.xml]: Do you need those """"...""""?

File: locale/fr_CA/editor.xml [hunk 2]
```diff
@@ -99,4 +99,40 @@
...
+	<message key="editor.submission.query.manageQueryNoteFilesDescription">Les fichiers peuvent être inclus dans cette discussion à partir de la liste ci-dessous.</message>
+	<message key="editor.submission.proofreading.confirmCompletion">Accepter la preuve pour indiquer que le document est prêt à être publié.</message>
+	<message key="editor.submission.proofreading.confirmRemoveCompletion">Retirer l'approbation de la preuve et indiquer que ce document n'est pas prêt pour la publication.</message>
+	<message key="editor.publicIdentificationExistsForTheSameType">Le public identifié comme '{$publicIdentifier}' existe déjà pour un autre objet de même type. Veuillez indiquer un public unique pour l'objet de même type à l'intérieur de votre revue.</message>
+	<message key="editor.publicIdentificationNumericNotAllowed">Le public identifié comme '{$publicIdentifier}' ne peut être un numéro.</message>
+	<message key="editor.publicIdentificationPatternNotAllowed">Le modèle {$pattern} ne peut pas être utilisé comme public identifié.</message>
+	<message key=""><![CDATA[]]></message>
```
Comment (mtub) [inline locale/fr_CA/editor.xml]: Please remove empty key...

File: locale/fr_CA/submission.xml [hunk 3]
```diff
@@ -559,4 +559,84 @@
...
+	<message key="submission.license.cc.by-sa4">Attribution - Partage dans les Mêmes Conditions 4.0 International (CC BY-SA 4.0)</message>
+	<message key="submission.license.cc.by-nc-nd4.footer"><![CDATA[<a rel="license" href="https://creativecommons.org/licenses/by-nc-nd/4.0/deed.fr"><img alt="Creative Commons License" src="//i.creativecommons.org/l/by-nc-nd/4.0/88x31.png" /></a><p>Ce travail est disponible sous licence <a rel="license" href="http://creativecommons.org/licenses/by-nc-nd/4.0/deed.fr">Creative Commons Attribution - Pas d'Utilisation Commerciale - Pas de Modification 4.0 International</a>.</p>]]></message>
+	<message key="submission.license.cc.by-nc4.footer"><![CDATA[<a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/deed.fr"><img alt="Creative Commons License" src="//i.creativecommons.org/l/by-nc/4.0/88x31.png" /></a><p>Ce travail est disponible sous licence <a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/deed.fr">Creative Commons Attribution - Pas d’Utilisation Commerciale 4.0 International</a>.</p>]]></message>
+	<message key=""><![CDATA[]]></message>
```
Comment (mtub) [inline locale/fr_CA/submission.xml]: Please remove empty key…

File: locale/fr_CA/submission.xml [hunk 4]
```diff
@@ -559,4 +559,84 @@
...
+	<message key="submission.license.cc.by-nc-sa4.footer"><![CDATA[<a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/4.0/deed.fr"><img alt="Creative Commons License" src="//i.creativecommons.org/l/by-nc-sa/4.0/88x31.png" /></a><p>Ce travail est disponible sous licence  <a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/4.0/deed.fr">Creative Commons Attribution - Pas d’Utilisation Commerciale - Partage dans les Mêmes Conditions 4.0 International</a>.</p>]]></message>
+	<message key=""><![CDATA[]]></message>
```
Comment (mtub) [inline locale/fr_CA/submission.xml]: Please remove empty key…

File: locale/fr_CA/submission.xml [hunk 5]
```diff
@@ -559,4 +559,84 @@
...
+	<message key="submission.license.cc.by-nd4.footer"><![CDATA[<a rel="license" href="http://creativecommons.org/licenses/by-nd/4.0/deed.fr"><img alt="Creative Commons License" src="//i.creativecommons.org/l/by-nd/4.0/88x31.png" /></a><p>Ce travail est disponible sous licence <a rel="license" href="http://creativecommons.org/licenses/by-nd/4.0/deed.fr">Creative Commons Attribution - Pas de Modification 4.0 International</a>.</p>]]></message>
+	<message key=""><![CDATA[]]></message>
```
Comment (mtub) [inline locale/fr_CA/submission.xml]: Please remove empty key…

File: locale/fr_CA/submission.xml [hunk 6]
```diff
@@ -559,4 +559,84 @@
...
+	<message key="submission.license.cc.by4.footer"><![CDATA[<a rel="license" href="http://creativecommons.org/licenses/by/4.0/deed.fr"><img alt="Creative Commons License" src="//i.creativecommons.org/l/by/4.0/88x31.png" /></a><p>Ce travail est disponible sous la licence  <a rel="license" href="http://creativecommons.org/licenses/by/4.0/deed.fr">Creative Commons Attribution 4.0 International </a>.</p>]]></message>
+	<message key=""><![CDATA[]]></message>
```
Comment (mtub) [inline locale/fr_CA/submission.xml]: Please remove empty key…

File: locale/fr_CA/submission.xml [hunk 7]
```diff
@@ -559,4 +559,84 @@
...
+	<message key="submission.license.cc.by-sa4.footer"><![CDATA[<a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/deed.fr"><img alt="Creative Commons License" src="//i.creativecommons.org/l/by-sa/4.0/88x31.png" /></a><p>Ce travail est disponible sous licence <a rel="license" href="http://creativecommons.org/licenses/by-sa/4.0/deed.fr">Creative Commons Attribution - Partage dans les Mêmes Conditions 4.0 International</a>.</p>]]></message>
+	<message key=""><![CDATA[]]></message>
```
Comment (mtub) [inline locale/fr_CA/submission.xml]: Please remove empty key…

File: plugins/generic/acron/locale/fr_CA/locale.xml [hunk 8]
```diff
@@ -14,7 +14,11 @@
  
 <locale name="fr_CA" full_name="Français (Canada)">
 	<message key="plugins.generic.acron.name">Plugiciel Acron</message>
+	<message key="plugins.generic.acron.description">Ce plugiciel tente de réduire la dépendance d'OCS aux outils de planificiation périodique tel que 'cron.'</message>
```
Comment (mtub) [inline plugins/generic/acron/locale/fr_CA/locale.xml]: This is not the most recent version of that key, please double-check the en_US version.

File: plugins/generic/usageStats/locale/fr_CA/locale.xml [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+	<message key="plugins.generic.usageStats.settings.dataPrivacyOption.excludesRegion">L'activation de la confidentialité des données exclut Region comme statistique facultative.</message>
+	<message key="plugins.generic.usageStats.settings.dataPrivacyCheckbox">Respect de la confidentialité des données</message>
+	<message key="plugins.generic.usageStats.settings.optionalColumns">Information statistique optionnel</message>
+	<message key="plugins.generic.usageStats.settings.optionalColumns.description">Activez ou désactivez la collecte des informations facultatives suivantes. Cela aura une influence sur les rapports statistiques possibles que vous pouvez récupérer et aura également un impact sur la taille de la base de données. NE PAS CHANGER, sauf si vous comprenez parfaitement ce que vous faites.</message>
+	<message key="plugins.generic.usageStats.settings.optionalColumns.cityRequiresRegion">Si vous cochez la colonne "Ville" vous devez cochez la colonne "Région"</message>
+	<message key="plugins.generic.usageStats.settings.archives">Archives</message>
+	<message key=""><![CDATA[]]></message>
```
Comment (mtub) [inline plugins/generic/usageStats/locale/fr_CA/locale.xml]: Please remove empty key…

File: plugins/generic/usageStats/locale/fr_CA/locale.xml [hunk 10]
```diff
@@ -0,0 +1,66 @@
...
+	<message key="plugins.generic.usageStats.settings.statsDisplayOptions.chartType">Choisissez le type du graphique pour afficher les statistiques de téléchargement</message>
+	<message key="plugins.generic.usageStats.settings.statsDisplayOptions.chartType.bar">Barre</message>
+	<message key="plugins.generic.usageStats.settings.statsDisplayOptions.chartType.line">Ligne</message>
+	<message key="plugins.generic.usageStats.settings.statsDisplayOptions.datasetMaxCount">Définissez le nombre maximal de données à présenter simultanément pour un point d'axe X spécifique. Une valeur plus élevée peut générer des graphiques difficiles à comprendre. Quelque chose entre 3 et 5 est sûr.</message>
+	<message key="plugins.generic.usageStats.usageStatsLoaderName">Statistiques sur le téléversement de fichier</message>
+	<message key="plugins.generic.usageStats.openFileFailed">Le fichier {$file} n'a pas pu être ouvert et a été rejeté.</message>
+	<message key="plugins.generic.usageStats.invalidLogEntry">La ligne {$lineNumber} du fichier {$file} n'est pas une entrée de journal valide. Le fichier a été rejeté.</message>
+	<message key=""><![CDATA[]]></message>
```
Comment (mtub) [inline plugins/generic/usageStats/locale/fr_CA/locale.xml]: Please remove empty key…

File: plugins/metadata/openurl10/locale/fr_CA/locale.xml [hunk 11]
```diff
@@ -0,0 +1,18 @@
...
+  * Copyright (c) 2014-2016 Simon Fraser University Library
+  * Copyright (c) 2003-2016 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Localization strings.
+  -->
+
+<locale name="fr_CA" full_name="Français (Canada)">
+	<message key="plugins.metadata.openurl10.displayName">Schéma de métadonnées OpenURL 1.0</message>
+	<message key=""><![CDATA[]]></message>
```
Comment (mtub) [inline plugins/metadata/openurl10/locale/fr_CA/locale.xml]: Please remove empty key…

File: plugins/metadata/nlm30/locale/fr_CA/locale.xml [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+	<message key="plugins.metadata.nlm30.citationLookup.displayName">Service pour les bases de données de citations</message>
+	<message key="plugins.metadata.nlm30.citationLookup.description">Améliore et corrige les éléments/balises de citation du NLM 3.0 en se connectant sur des bases de données externes.</message>
+	<message key="plugins.metadata.nlm30.citationOutput.displayName">Service de d'extraction de citations</message>
+	<message key="plugins.metadata.nlm30.citationOutput.description">Convertis une citation en balises/éléments du schéma NLM 3.0 en format standard plein texte.</message>
+	<message key=""><![CDATA[]]></message>
+	<message key="plugins.metadata.nlm30Xml.citationOutput.displayName">Service d'extraction de citations (XML)</message>
+	<message key="plugins.metadata.nlm30Xml.citationOutput.description">Converti une citation en balises/éléments du schéma NLM 3.0 en format standard XML.</message>
+	<message key="plugins.metadata.nlm30.nlm30XmlOutput.displayName">Format de publication de revue NLM 3.0</message>
+	<message key=""><![CDATA[]]></message>
```
Comment (mtub) [inline plugins/metadata/nlm30/locale/fr_CA/locale.xml]: Please remove empty key…

File: plugins/metadata/nlm30/locale/fr_CA/locale.xml [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+	<message key="plugins.metadata.nlm30.nlm30XmlOutput.description">Converti une soumission au format NLM 3.0</message>
+	<message key="plugins.metadata.nlm30.nlm23XmlOutput.displayName">Format de publication de revue NLM 2.3</message>
+	<message key=""><![CDATA[]]></message>
```
Comment (mtub) [inline plugins/metadata/nlm30/locale/fr_CA/locale.xml]: Please remove empty key…

**Comment by pnault** [comment]
Please ignore commit "removing empty keys and one variable according to master version". This is an error !

**Comment by pnault** [comment]
Hi Marco,
I have made all changes to the the files you requested. You can start reviewing it.

Pierre

**Comment by mtub** [comment]
Thank you so much!

**Pull request merged**